.ci/windows/msbuild-tools.ps1

```diff
@@ -3,6 +3,6 @@
 #
 
 # Install visualstudio2019buildtools
-& choco install visualstudio2019buildtools -m -y --no-progress --force -r --version=16.4.5.0
-& choco install visualstudio2019professional -m -y --no-progress --force -r --version=16.4.5.0 --package-parameters "--includeRecommended --includeOptional"
+& choco install visualstudio2019buildtools -m -y --no-progress --force -r --version=16.7.7.0
+& choco install visualstudio2019professional -m -y --no-progress --force -r --version=16.7.7.0 --package-parameters "--includeRecommended --includeOptional"
```

Comment:
See:
- https://chocolatey.org/packages/visualstudio2019buildtools/16.7.7.0
- https://chocolatey.org/packages/visualstudio2019professional/16.7.7.0

Latest msbuild version is 16.7.0 -> https://github.com/dotnet/msbuild/releases